go.mod

```diff
@@ -45,6 +46,7 @@ require (
 	github.com/go-redis/redis v6.15.2+incompatible
 	github.com/go-sql-driver/mysql v1.4.1
 	github.com/go-swagger/go-swagger v0.21.0
+	github.com/go-xorm/xorm v0.7.9
```

Comment:
why do you use a old xorm ?

-> xorm.io/xorm v0.8.2-0.20200120024500-c37aff9b3a4a

Reply:
Merge issues.. Will resolve